dio/lib/dio.dart

```diff
@@ -5,6 +5,8 @@
 /// {@category Plugins}
 library dio;
 
+export 'package:http_parser/http_parser.dart' show MediaType;
```

Comment:
However, exporting the `MediaType` from here seems a bad part for me. I'd prefer to export it from the `multipart_file.dart`.